MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,364 @@
+# Migration Guide
+
+## Migrate from OpenTracing to OpenTelemetry 
+
+### Starter Bundle
+If you do not use sda-commons-starter with [SdaPlatformBundle](../../sda-commons-starter/src/main/java/org/sdase/commons/starter/SdaPlatformBundle.java), you need to remove the Jaeger bundle and OpenTracing bundle and add the OpenTelemetry bundle.
+
+Before:
+```java
+bootstrap.addBundle(JaegerBundle.builder().build());
+bootstrap.addBundle(OpenTracingBundle.builder().build());
+```
+
+After:
+```java
+bootstrap.addBundle(
+        OpenTelemetryBundle.builder()
+            .withAutoConfiguredTelemetryInstance()
+            .withExcludedUrlsPattern(Pattern.compile(String.join("|", excludedTracingUrls)))
```

Comment:
@elbiocaetano : Is that mandatory? The setup now looks more complicated than for v2. What is the value of `excludedTracingUrls`?

Reply:
This is a list of excluded urls, e.g: healthcheck and metrics. it is not mandatory, but I liked to leave there, because I thought it would be useful when configuring manually. Also, in the dw3 it automatically excludes some patterns of urls, so to be the same behavior they should do the same in their app. Do you think I can put in the code the declaration of excludedTracingUrls?

Reply:
What about adding the value that we use in the `SdaPlatformBundle`? Since that value was not needed in v2 people would not know what to use here.